### ISSUE TITLE
Platform support for iOS

### ISSUE THREAD

**Issue by Prismic247** [JordoVR/EasyQuestSwitch#35]
The VRChat SDK now supports building and uploading worlds and avatars for iOS. While it is "EasyQuestSwitch", are there plans to add iOS support as well?

**Comment by TheXev** [comment]
I would love to see support for iOS added as well. If there were an option to simply allow iOS build target to = Android most of the time, that'd be the best method imo, as most of the time you are wanting to do the same things in iOS and Android... at least in my own experience.

**Comment by ghost** [comment]
With the release of the SDK's multi-platform upload; we really need this. I wouldn't mind if it used the same settings as the "Quest" settings

**Comment by MarcellaVT** [comment]
There is a pull request in this repo from [bdragon28's fork](https://github.com/bdragon28/EasyQuestSwitch/tree/pr-32) that adds iOS support. Its been sitting as a pull request for over a year, in my testing of that fork it works just fine. I wanted to start supporting iOS for my group but it was really annoying to have to manually install that fork especially with a team of people working on the world. So we made another fork off of bdragon28's and made a vcc listing for it. We ended up renaming it to [EasyPlatformSwitch](https://github.com/VRC-Kmart/EasyPlatformSwitch) so its easier to distinguish from original and also because i felt like the old name wasnt the most accurate anymore.

Tldr: I made a fork of a fork of a verion of EasyQuestSwitch that adds iOS support and made a vcc package for my group and other people to use.

https://vrc-kmart.github.io/VRCKmartVCCListing/
